### PR TITLE
Precipitation Probability

### PR DESCRIPTION
**Pull request by Mackenzie-Frey**
_Completes_
- If precip probability is less than 10%, the icon will not display.
- Enhances testing to test for each icon.
- Removes display flex on the dashboard to return the page to previous mobile compatibility functionality.

sloses#171

User Story:
https://waffle.io/le3ah/thirsty_plants/cards/5c6f2a7c93169e001bddae69

### CODE REVIEW

File: app/models/weather.rb [hunk 1]
```diff
@@ -32,11 +32,17 @@ def precip_type(day_index)
...
       icon = 'far fa-snowflake'
-    elsif type == 'sleet'
+    elsif type == 'sleet' && chance_of_rain(day_index) >= 10
       icon = 'fas fa-cloud-showers-heavy'
+    elsif type == 'rain' && chance_of_rain(day_index) >= 10
+      icon = 'fas fa-cloud-rain' 
+    elsif type == nil && chance_of_rain(day_index) >= 10
+      icon = 'fas fa-cloud-rain'
+    else
+      icon = ""
```
Comment (le3ah) [inline app/models/weather.rb]: Great!

File: spec/features/users/users_can_see_see_precip_icons_spec.rb [hunk 2]
```diff
@@ -15,7 +15,7 @@
                 "summary" => "Heavy snow (2–5 in.) starting in the evening.",
                 "icon" => "snow",
                 "precipIntensity" => 0.0038,
-                "precipProbability" => 0.43,
+                "precipProbability" => 12.0,
```
Comment (le3ah) [inline spec/features/users/users_can_see_see_precip_icons_spec.rb]: ☃️

File: spec/features/users/users_can_see_see_precip_icons_spec.rb [hunk 3]
```diff
@@ -32,4 +32,126 @@
...
+
+    garden = create(:garden, lat: lat, long: long)
+    plant = create(:plant, garden: garden)
+
+    weather = Weather.new(garden)
+    allow(weather).to receive(:weather_info).and_return(weather_info)
+    precip_type = weather.precip_type(day_index).capitalize
+    precip_icon = weather.precip_icon(day_index)
+
+    expect(precip_type).to eq("Rain")
```
Comment (le3ah) [inline spec/features/users/users_can_see_see_precip_icons_spec.rb]: Great job testing! 🌧

**Comment by abenetka** [comment]
Thanks for making these changes!!

**Pull request merged**